app/models/friend.rb

```diff
@@ -58,7 +55,18 @@ def validate_address_with_smartystreets
 
         first_candidate = result[0]
 
-        puts "There is at least one candidate.\n\nThe match parameter IS set to STRICT, this means the address IS valid.\n\n"
+        if lookup.match == :strict
+            puts "There is at least one candidate.\n\nThe match parameter IS set to " + lookup.match.to_s + ", this means the address IS valid.\n\n"
+        elsif lookup.match == :exact
+            puts "There is at least one candidate.\n\nThe match parameter IS set to " + lookup.match.to_s + ", this indicates an exact match.\n\n" 
+        elsif lookup.match == :range
+            puts "There is at least one candidate.\n\nThe match parameter IS set to " + lookup.match.to_s + ", this Indicates a match within a range of addresses.\n\n" 
+        elsif lookup.match == :invalid
+            puts "There is at least one candidate.\n\nThe match parameter IS set to " + lookup.match.to_s + ", this means the addrress may or may not be valid and you need to perform further validations.\n\n"
+        else
+            puts "Invalid match type" + "\n\n"
+        end
+
```

Comment:
Cruft. This is not needed. If you need to know wether an address is as you expect it, return an error or a success.